### PR TITLE
LP-2976 Send account activation messages in context

### PR DESCRIPTION
**Pull request by asadali145**
[LP-2976](https://philanthropyu.atlassian.net/browse/LP-2976)

***

#### Description:

- Show account activation messages in new dashboard

***

## Things to Cross-check:

- [x] Technical Docs updated if needed?
- [x] [Domain Model](https://drive.google.com/file/d/1iRSqxDQQMXl6JzLw9nZtq1yGshwMBLTn/view?usp=sharing) updated if needed?
- [x] [Core Changes Wiki](https://github.com/OmnipreneurshipAcademy/edx-platform/wiki/Core-Changes) updated if needed?

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3566/workflows/workflows_id/jobs/3566/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (90.0%): Missing lines 143
-------------
Total:   12 lines
Missing: 1 line
Coverage: 91%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3566/workflows/workflows_id/jobs/3566/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (90.0%): Missing lines 143
-------------
Total:   12 lines
Missing: 1 line
Coverage: 91%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3566/workflows/workflows_id/jobs/3566/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (90.0%): Missing lines 143
-------------
Total:   12 lines
Missing: 1 line
Coverage: 91%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3566/workflows/workflows_id/jobs/3566/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (90.0%): Missing lines 143
-------------
Total:   12 lines
Missing: 1 line
Coverage: 91%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3566/workflows/workflows_id/jobs/3566/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (90.0%): Missing lines 143
-------------
Total:   12 lines
Missing: 1 line
Coverage: 91%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3566/workflows/workflows_id/jobs/3566/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (90.0%): Missing lines 143
-------------
Total:   12 lines
Missing: 1 line
Coverage: 91%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3566/workflows/workflows_id/jobs/3566/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (90.0%): Missing lines 143
-------------
Total:   12 lines
Missing: 1 line
Coverage: 91%
-------------
```

**Review by Syed-Ali-Abbas-Zaidi — APPROVED** [review]

File: openedx/adg/lms/student/views.py [hunk 1]
```diff
@@ -124,10 +128,37 @@ def get_context_data(self, **kwargs):
         """
         context = super(DashboardView, self).get_context_data(**kwargs)
 
+        platform_name = configuration_helpers.get_value('platform_name', settings.PLATFORM_NAME)
+        activation_email_support_link = configuration_helpers.get_value(
+            'ACTIVATION_EMAIL_SUPPORT_LINK', settings.ACTIVATION_EMAIL_SUPPORT_LINK
+        ) or settings.SUPPORT_SITE_LINK
```
Comment (annas-usman) [inline openedx/adg/lms/student/views.py:134]: We can move these into the if condition block as these variables are only being used inside the if condition block.

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3567/workflows/workflows_id/jobs/3567/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (70.0%): Missing lines 138-139,143
-------------
Total:   12 lines
Missing: 3 lines
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3567/workflows/workflows_id/jobs/3567/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (70.0%): Missing lines 138-139,143
-------------
Total:   12 lines
Missing: 3 lines
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3567/workflows/workflows_id/jobs/3567/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (70.0%): Missing lines 138-139,143
-------------
Total:   12 lines
Missing: 3 lines
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3567/workflows/workflows_id/jobs/3567/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (70.0%): Missing lines 138-139,143
-------------
Total:   12 lines
Missing: 3 lines
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3567/workflows/workflows_id/jobs/3567/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (70.0%): Missing lines 138-139,143
-------------
Total:   12 lines
Missing: 3 lines
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3567/workflows/workflows_id/jobs/3567/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (70.0%): Missing lines 138-139,143
-------------
Total:   12 lines
Missing: 3 lines
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3567/workflows/workflows_id/jobs/3567/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (70.0%): Missing lines 138-139,143
-------------
Total:   12 lines
Missing: 3 lines
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3568/workflows/workflows_id/jobs/3568/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (100%)
-------------
Total:   13 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3568/workflows/workflows_id/jobs/3568/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (100%)
-------------
Total:   13 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3568/workflows/workflows_id/jobs/3568/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (100%)
-------------
Total:   13 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3568/workflows/workflows_id/jobs/3568/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (100%)
-------------
Total:   13 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3568/workflows/workflows_id/jobs/3568/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (100%)
-------------
Total:   13 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3568/workflows/workflows_id/jobs/3568/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (100%)
-------------
Total:   13 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3568/workflows/workflows_id/jobs/3568/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/student/tests/test_views.py (100%)
openedx/adg/lms/student/views.py (100%)
-------------
Total:   13 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by annas-usman — APPROVED** [review]

**Pull request merged**